actgen/train.py

```diff
@@ -128,7 +131,14 @@ def evaluate(self, step):
             is_best = True
         else:
             is_best = False
-        self.agent.save(is_best, self.params['seed'])
+        regularizer = self.params['regularization']
+        if 'l1' in self.params['regularization']:
+            regularizer += "_" + str(self.params['regularization_weight_l1'])
+        if 'l2' in self.params['regularization']:
+            regularizer += "_" + str(self.params['regularization_weight_l2'])
+        if 'dropout' in self.params['regularization']:
+            regularizer += "_" + str(self.params['dropout_rate'])
+        self.agent.save(is_best, self.params['seed'], regularizer)
```

Comment:
I think it's time to upgrade the organization of the results folder. We can leave this as-is for now, but we should plan on revisiting the way results files are saved soon. Rather than a flat folder, I think we can have a `--tag` for each experiment, and then organize results by experiment

Reply:
I can change this now, together with the previous comment about saving results. 

Is the idea just to save the model under `results/tag_name/model_file_name.torch`? 

To make this compatible with onager, does is matter if the flag is called `--tag` or `--run_tag`?

Reply:
Either will work. The `onager prelaunch` command only parses arguments with a `+` prefix, so the `--run_tag` in the example is an argument that the worker script understands. I personally prefer `--tag`, since it's easier to type

Reply:
To answer the first question, I was hoping to have a two-level hierarchy. The top-level will be `--results_dir`, which will default to the current `./results/` directory, then the next level is the `--tag` directory. This way, if we later change our minds and need a 3-level structure, we can do something like `--results_dir='./results/sub-experiment/'`